### ISSUE TITLE
How does UniWorld perform on other editing benchmarks?

### ISSUE THREAD

**Issue by Andy1621** [PKU-YuanGroup/UniWorld-V1#6]
Thank you for sharing this interesting work!
After reviewing the data and performance metrics, I’m particularly curious about UniWorld’s results on other editing benchmarks, such as [GEdit-Bench](https://github.com/stepfun-ai/Step1X-Edit/tree/main/GEdit-Bench) used in Step1X-Edit. From what I can see, the ImgEdit benchmark appears to be in-domain with the training data ([reference](https://huggingface.co/datasets/LanguageBind/UniWorld-V1/tree/main/data/imgedit)), so I wonder how Uniworld would perform on a more out-of-domain or challenging benchmark.

In my try, the out-of-domain editing is not satisfactory.

<img width="400" alt="Image" src="https://github.com/user-attachments/assets/cfee3f03-c845-4a6e-bdae-989d61a157ed" />

For text editing, the man's face is changed, prompt "write the text "gucci" to the bag":

<img width="600" alt="Image" src="https://github.com/user-attachments/assets/11a30866-f7fb-45c2-96c2-cc1d4704054d" />

Furthermore, the results of complicated prompt or motion manipulation are not satisfactory, too.

<img width="600" alt="Image" src="https://github.com/user-attachments/assets/2e05b5f8-327b-4c95-abb9-3ded06fe13cc" />

**Comment by LinB203** [comment]
As noted in the Limitations section of our report, this issue stems from the fact that most of our editing data is drawn from ImgEdit, resulting in lower instruction diversity than BAGEL or Step1X-Edit. We also evaluated GEdit. However, its instruction score showed substantial variance (due to seed or prompt effects), despite yielding high perceptual scores.

We infer that this can be addressed by incorporating additional data and fine-tuning the VLM. Our core contribution is the finding that high-resolution SigLIP enables control over reference-image consistency.

**Comment by LinB203** [comment]
Notably, our dataset contains no text‐editing samples, leading to poor performance on such tasks.

**Comment by causalfusion** [comment]
> As noted in the Limitations section of our report, this issue stems from the fact that most of our editing data is drawn from ImgEdit, resulting in lower instruction diversity than BAGEL or Step1X-Edit. We also evaluated GEdit. However, its instruction score showed substantial variance (due to seed or prompt effects), despite yielding high perceptual scores.
> 
> We infer that this can be addressed by incorporating additional data and fine-tuning the VLM. Our core contribution is the finding that high-resolution SigLIP enables control over reference-image consistency.

Thank you very much for clarifying this. It would be better if you can also clarify this in your report.  At least the readers should be aware that UniWorld has used in-domain data from ImgEdit, and cannot perform well on other edit benchmarks.

**Comment by Andy1621** [comment]
Thanks for yopur reply. Provide two more bad cases for future improvement:

(1) Do not support Chinese

<img width="1247" alt="Image" src="https://github.com/user-attachments/assets/a3eaf58f-2519-453b-b0e4-f0ef92164b0d" />

(2) Style transfer is strange (Change to Pixar style):

<img width="941" alt="Image" src="https://github.com/user-attachments/assets/334e3574-aed5-4864-aebd-b7251c111a0f" />

**Comment by causalfusion** [comment]
> We will re-evaluate and include the updates in the next revision.

Thanks!

Also I am interested in the observation

> Our core contribution is the finding that high-resolution SigLIP enables control over reference-image consistency.

However, in the "Einstein holding Gucci bag" example, some details from the reference image—such as the hair and facial expression of the main character—appear to have changed.
Do you think this can still be addressed by increasing the input resolution of SigLIP?

**Comment by causalfusion** [comment]
> Hi, everyone, we have evaluated the UniWorld-V1 on GEdit-Bench. The results will be updated on technical report tonight. The results and descriptions are shown in attached figure. As noted above, the absence of domain-specific data (e.g., text_change) substantially depresses the scores.
> 

Cool! Btw, could you also update the claims in the paper, such as “using 0.1% of the data and outperforming BAGEL on image editing”? As it stands, this could be misleading to readers, since the evaluation was conducted on ImgEdit, where your model was trained on in-domain data. The performance gain may largely stem from this setup rather than from superior generalization.

**Comment by LinB203** [comment]
> > Hi, everyone, we have evaluated the UniWorld-V1 on GEdit-Bench. The results will be updated on technical report tonight. The results and descriptions are shown in attached figure. As noted above, the absence of domain-specific data (e.g., text_change) substantially depresses the scores.
> 
> Cool! Btw, could you also update the claims in the paper, such as “using 0.1% of the data and outperforming BAGEL on image editing”? As it stands, this could be misleading to readers, since the evaluation was conducted on ImgEdit, where your model was trained on in-domain data. The performance gain may largely stem from this setup rather than from superior generalization.

Yes, please don't worry. When we decide to update the result, we will definitely modify the description simultaneously.

**Comment by LinB203** [comment]
@Andy1621 @causalfusion Hello everyone, we have updated the abstract, introduction, and corresponding experiments on [arXiv](https://arxiv.org/abs/2506.03147) last night. Feel free to contact me if you have any questions.
